zpay32/fuzz_test.go

```diff
@@ -12,15 +12,7 @@ import (
 
 func FuzzDecode(f *testing.F) {
 	f.Fuzz(func(t *testing.T, data string) {
-		inv, err := Decode(data, &chaincfg.TestNet3Params)
-		if err != nil {
-			return
-		}
-
-		// Call these functions as a sanity check to make sure the
-		// invoice is well-formed.
-		_ = inv.MinFinalCLTVExpiry()
```

Comment:
I think this should stay since it can let us know if the function breaks

Reply:
We can get rid of it, though sometimes I find this pattern useful to see if a member function gives an unexpected side effect